helix-term/src/commands.rs

```diff
@@ -2928,7 +2978,21 @@ pub mod cmd {
             doc: "Go to line number.",
             fun: goto_line_number,
             completer: None,
-        }
+        },
+        TypableCommand {
+            name: "sort",
+            aliases: &[],
+            doc: "Sort lines in selection.",
+            fun: sort,
+            completer: None,
+        },
+        TypableCommand {
+            name: "sort!",
```

Comment:
Also, maybe call this `rsort` instead of `sort!`?

Reply:
`rsort` makes more sense to me personally.

Reply:
For sure

Reply:
Renamed to `rsort`.